tests/integration/smoke_test.go

```diff
@@ -81,6 +81,10 @@ func (suite *SmokeSuite) TearDownSuite() {
 	suite.op.Teardown()
 }
 
+func (suite *SmokeSuite) TestACephCSI_SmokeTest() {
```

Comment:
@rootfs failing to interpret `TestACephCSI" in the function name . Do we really need "A" in this func name ?

Reply:
that letter bumps csi test to be the first one to run, so i don't have to wait :)

will remove that letter before merge

Reply:
name changed